helm-hooks/postinstall/postinstall.go

```diff
@@ -55,6 +55,9 @@ webhooks:
         resources:
           - pods
     failurePolicy: Fail
+    namespaceSelector:
+      matchExpressions:
+      - {key: kritis-validation, operator: NotIn, values: [disabled]}
```

Comment:
I also thought about using key like: `kritis-disabled: true` but then it can create confusion with true being string or bool. So, I do not want to use true in label.